frontend/static/js/pages/play.js

```diff
@@ -209,6 +231,13 @@ let app = new Vue({
             if (this.path.length == 0 || this.path[this.path.length - 1]["article"] != page) {
                 // Update path
                 let timeElapsed = (Date.now() - this.startTime) / 1000;
+		
```

Comment:
We could probably do something like 
```
 if (this.isPenaltyMode) {
                    this.penaltyTime += 20;
		        }

                this.path.push({
                    "article": page,
                    "timeReached": timeElapsed,
                    "loadTime": loadTimeSeconds,
                    "penaltyTime": penaltyTime,

                });
```

Reply:
Okay that makes sense, I’ll look more into that.

Reply:
Also, for the leaderboard, I made a new option for penalty mode. I was thinking of adding a new column “time (w penalty)” and the penalty mode option for leaderboard will sort by that column. 

Just to clarify is this the approach we’re looking for?

Reply:
Yeah! I think that's what @dqian3 was talking about